symlink/doc.go

```diff
@@ -1,4 +1,11 @@
-// Package symlink implements EvalSymlinksInScope which is an extension of
-// filepath.EvalSymlinks, as well as a Windows long-path aware version of
-// filepath.EvalSymlinks from the Go standard library (https://golang.org/pkg/path/filepath).
+// Package symlink implements [FollowSymlinkInScope] which is an extension
+// of [path/filepath.EvalSymlinks], as well as a Windows long-path aware
+// version of [path/filepath.EvalSymlinks] from the Go standard library.
+//
+// The code from [path/filepath.EvalSymlinks] has been adapted in fs.go.
+// Read the [LICENSE.BSD] file that governs fs.go and [LICENSE.APACHE] for
+// fs_unix_test.go.
```

Comment:
Can/should we add references to the files we're talking about (i.e. `fs.go` -> `[fs.go]`)?

Reply:
I was considering that initially, but didn't know what the most suitable link for that would be (linking to `main` could cause the link to break potentially, and linking to a versioned one could be confusing when linking from version X to version "old"), so I "somewhat" kept that an exercise to the user for now.